database/migration/V20170126_1948__createCheckInTable.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE `checkins` (
```

Comment:
The date of this migration seems too early to work in the migration chain. Did you run the dev migrations?